config/jobs/kubernetes-sigs/sig-windows/soak-tests.yaml

```diff
@@ -21,7 +21,7 @@ presubmits:
       #   path_alias: "k8s.io/perf-tests"
       - org: kubernetes-sigs
         repo: cluster-api-provider-azure
-        base_ref: main
+        base_ref: release-1.3
```

Comment:
@marosset @oprinmarius - I don't think we are using anything from the main branch but fyi